### DISCUSSION TITLE
2 unexpected ELRS failsafe with good RSSI dbm and LQ

### DISCUSSION THREAD

**Discussion by AnselmeCFPV** [ExpressLRS/ExpressLRS#2679] [Q&A]
Hello !

I'm using a Nano ELRS RX, a Taranis X9D+ with a R9M emetter, both on 915Mhz. I never had any range issue with my 5" drone which I built recently (it's my 20th battery approximately). But today, I had two brutal and unexpected failsafes with good RSSI dbm and link quality. It just came out of nowhere a few meters of me.

When I picked up the drone, it didn't get signal back (don't know if it's supposed to actually or you have to power cycle), it was almost like the TX and RX unbound themselves midflight. I powered cycle the drone, without success. I powered cycle the Taranis and the drone, and this succeeded. I went for a second cautious flight, almost nothing, and when I came in to land, boom, second failsafe.

You can watch this small private video to see what happened on the DVR, maybe it will help : https://www.youtube.com/watch?v=2uhNcKQDeBc
(Just know that my failsafe is configured to put the drone level and climb a few meters before dropping).
100m of my take off point there is a high speed train track, but I don't think that the cable could cause any interference at this distance... 

As I never had this issue with longer flights and really good signal (500m+) I am really worried that this might happen in the future, and the worst is I don't know what's the case : RX ? TX ? Flight Controller ?

Thank you for any help that you could bring !

**Comment by JyeSmith** [comment]
Since the LQ is counting down this looks like a TX side issue.

The R9 is known to over heat and have issues without cooling.  What power are you running?

And at leat one of my R9 modules has problem maintaining a good connection in the JR bay.  Give yours a good shake and check it doesn't turn off or power cycle.

**Comment by AnselmeCFPV** [comment]
Thank you for your answer. I'm on 100mW emission power which to me isn't that much, is it ? The emitter is not hot or anything, and it was a bit chill outside. Regarding the JR bay connexion, it is alright, even when moving the TX it has good connexion no matter what.
So I don't understand where the problem is from. My RX wiring is good. I can't reproduce the problem on ground.

Regarding the connection, if it had powered cycle I would have heard it. And after drone crashed I immediately looked at my R9M and it was definitely powered up. LQ dropping to 0 suddenly is very scary. Couldn't be interference or is it very unlikely ? 

Thank you for your help

**Reply by Trumblefly** [reply]
Just want to add that I just had the exact same problem with my Wing.
3 Failsaves in a row, 3 launches where it failsaved just before my person launching would've thrown it.
For a launch he is always running about 15m and when he's about 3m away from me he throws it.
I have a frsky taranis qx7s with edgetx and a elrs module on dynamic power.
I think it might have had something to to with me changing the hz from 150 to 100 and the telemetry from 1:32 to 1:2 before the 3 failsaves, im not sure at all tho.

**Reply by AnselmeCFPV** [reply]
Hello, have you tried disabling dynamic power ? While doing researches about this problem I saw this advice. Try running on a static power and see if it solves the problem ? Because if the telemetry parameters changed, then maybe the TX is not able to correctly adjust its power.

**Reply by Trumblefly** [reply]
I have not tried turning of dynamic power yet, thank you for the advice.
What I can say now tho is that it was not caused by the refresh rate I tried that and had the same results.
I will try turning of dynamic power soon and I am very happy about any other advice!

**Reply by Trumblefly** [reply]
I just wanted to write that I can't recreate the issue either (I have not testet flying again but i've basically recreated the launches but without a prop on.)
That means I can't try if turning of Dynamic power will help since it doesn't even failsave with it on.
Maybe its just a thing that happens on one day and then never again.
Just for safety i will update my elrs module and reciever, maybe its a bug in software, on what elrs version are you?

**Reply by AnselmeCFPV** [reply]
I run 3.3.2 if I remember well which is the last stable version. Upgrading to the latest version is recommended I think, so you should try it.

**Comment by AnselmeCFPV** [comment]
Hello, just to bring a quick update.

I left my Taranis and R9M powered on my bench with 100mW setting. The top left indicator on the ELRS Lua Script was showing (0/200) which is good. 5 minutes after, suddenly this indicator went down to something like 40/80 which is of course really bad. So it is likely a overheating issue. So I suggest to do the fan mod when you want to go above 50mW. https://www.expresslrs.org/hardware/fan-mod/
